app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -22,20 +19,30 @@ class Api::V2::SubscriptionsController < Api::V2::ApiController
     api :GET, "/organizations/:organization_id/subscriptions", N_("List organization subscriptions")
     api :GET, "/systems/:system_id/subscriptions", N_("List a system's subscriptions"), :deprecated => true
     api :GET, "/activation_keys/:activation_key_id/subscriptions", N_("List an activation key's subscriptions")
+    api :GET, "/subscriptions"
+    param_group :search, Api::V2::ApiController
     param :organization_id, :number, :desc => N_("Organization ID"), :required => true
     param :system_id, String, :desc => N_("UUID of the system"), :required => false
```

Comment:
I think it makes sense to call everything content host as you are suggesting, the two names confuse me and I am sure confuse users. I will make the change here and in the UI